src/LSP12/authenticateIssuedAssets/authenticateIssuedAssets.test.ts

```diff
@@ -0,0 +1,260 @@
+import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
+import { Signer, keccak256, toUtf8Bytes } from 'ethers';
+import { ethers } from 'hardhat';
```

Comment:
Is there a reason you are using:
- some of the ethers library functions from `ethers`
- using `ethers` from Hardhat?